Automattic/wp-calypso
# Stats: Add Mobile Apps promo card via DotPager UI.

#### Proposed Changes

Adds the Jetpack mobile apps promo card alongside the Yoast SEO one using the `DotPager` component. The Yoast card is still the default.

Before:

![APromoSEO](https://user-images.githubusercontent.com/40267301/202101081-db395e83-8d1a-4208-8bc1-8654833efe99.png)

After:

![NewPromoCards](https://user-images.githubusercontent.com/40267301/203266516-9b427729-7a0b-437f-9bd6-3b22ecbed058.png)

#### Testing Instructions



1. Visit the Calypso Live link.
2. Visit the Stats page.
3. Scroll down and confirm the `DotPager` card is present.
4. Confirm the Yoast SEO card is the default. 
5. Confirm the mobile apps card is available on the 2nd page.
6. Confirm it behaves appropriately on smaller screen sizes.

#### Pre-merge Checklist





Related to #69230 and supersedes #70059.

Currently just updating the Stats page. The designs call for the same promo card on the Insights and Ads pages but it looks overpowering there with the older page layout. I'm thinking we should wait until we "grid" those layouts before adding this component in.
